new/LIT5901DersanaGabrielNaglon.xml

```diff
@@ -0,0 +1,83 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5901DersanaGabrielNaglon" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+              <title xml:lang="en" corresp="#t1">Homily on construction of a church of Archangel Gabriel at Naglon (Naqlūn)</title>
+              <title xml:lang="gez" xml:id="t2">ድርሳነ፡ ገብርኤል፡</title>
+              <title xml:lang="gez" corresp="#t2" type="normalized">Dǝrsāna Gabrǝʾel</title>
+              <title xml:lang="en" corresp="#t2">Homily on Archangel Gabriel</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p>Anonymous homily telling a story of construction of a church of Archangel Gabriel
+                  at Naqlūn (here spelled <foreign xml:lang="gez">Naglon</foreign>), similar to
+                  <ref type="work" corresp="LIT3956Homily">the homily ascribed to Bishop of <placeName ref="LOC3066Fayyum"/>
+                   Alexander Abawre, a part of <ref type="work" corresp="LIT1284Dersan">Dǝrsāna Gabrǝʾel</ref>, as in
```

Comment:
I don't understand Alexander Abawre...LIT3956Homily is ascribed to Sawiros?

Reply:
No, I have to correct. Sawiros appears in only one ms, it may be a corruption of "Alexander Abawre". It took me several hours to understand this.